app/controllers/users_controller.rb

```diff
@@ -66,7 +65,7 @@ def destroy
   end
 
   def add_status
-    unless params[:user][:status].blank?
```

Comment:
@tansaku we cannot ask for `:status` if we don't have `:user` and this was failing. I created a method for working the same logic, what do you think?